docs/source/data-access-api/index.rst

```diff
@@ -16,21 +16,201 @@ that the full implementation of the functionality is yet to be completed.*
 
 The API can be accessed from the following URL [WIP]: https://api.dev.imap-mission.com
 
-.. openapi:: openapi.yml
-   :group:
-   :include: /upload
+Command Line Utility
+--------------------
+To Install
+^^^^^^^^^^
+
+Run the following command to use the API CLI:
+
+    .. code-block:: bash
+
+        pip install imap-data-access
+
+Base Command Arguments
+^^^^^^^^^^^^^^^^^^^^^^
+
+The following are base command arguments for the CLI:
+
+    .. code-block:: bash
+
+        imap-data-access -h # or
+        imap-data-access query # or
+        imap-data-access download # or
+        imap-data-access upload
+
+Add the -h flag with any base command for more information on use and functionality.
+
+Query
+^^^^^
+
+To query for files, you can use several parameters: ``--instrument``, ``--data-level``, ``--descriptor``, etc.
+
+Further information is found in in the ``query -h`` menu. You can use parameters alone, or in combination.
+
+**Example Usage:**
+
+    .. code-block:: bash
+
+        imap-data-access query --start-date 20240101 --end-date 20241231 --output-format json
+        # The following line is returned:
+        [{'file_path': 'imap/swe/l0/2024/01/imap_swe_l0_sci_20240105_v001.pkts', 'instrument': 'swe', 'data_level': 'l0', 'descriptor': 'sci', 'start_date': '20240105', 'version': 'v001', 'extension': 'pkts'}, {'file_path': 'imap/swe/l0/2024/01/imap_swe_l0_sci_20240105_v001.pkts', 'instrument': 'swe', 'data_level': 'l0', 'descriptor': 'sci', 'start_date': '20240105', 'version': 'v001', 'extension': 'pkts'}]
```

Comment:
Could you split this up into multiple lines? The way this is displayed on the page (one long line that you have to scroll through) is a bit difficult to see.